### PR TITLE
LGTM: Recommended JavaDoc fixes, header updates, whitespace removal.

### PR DESCRIPTION
**Pull request by rjhancock**
Just recommended updates from LGTM related to JavaDoc, 1 space added at the end of a single sentence for readability, updated copyright headers on the files, whitespace removal.

### CODE REVIEW

**Comment by HammerGS** [comment]
This pull request **fixes 102 alerts** when merging 252aedd30b1c6dc29da94dabc5f30cc918173425 into 54180d578d581886930b45341580a188d2fdeb2e - [view on LGTM.com](https://lgtm.com/projects/g/MegaMek/megamek/rev/pr-103d5e8e3d740fad08e212930bf940c3711a7bdd)

**fixed alerts:**

* 101 for Spurious Javadoc @param tags
* 1 for Missing space in string literal

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by neoancient** [comment]
I am not a lawyer, but I don't think we can remove somebody's copyright statement and replace it with a different copyright holder.

**Comment by rjhancock** [comment]
Corrected and combined the copyrights, corrected a few misspellings.

**Comment by HammerGS** [comment]
This pull request **fixes 102 alerts** when merging 507f810d872cb5aa899069203a942316a8c944e9 into 54180d578d581886930b45341580a188d2fdeb2e - [view on LGTM.com](https://lgtm.com/projects/g/MegaMek/megamek/rev/pr-eb1cc7bbf3a5189bbe95715fd04107e2df8599f0)

**fixed alerts:**

* 101 for Spurious Javadoc @param tags
* 1 for Missing space in string literal

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: megamek/src/megamek/common/Bay.java [hunk 1]
```diff
@@ -497,14 +500,13 @@ public int getFacing() {
     /**
      * Sets the armor facing for the bay, if the bay type requires it. If not required by the bay
      * type, does nothing.
-     * 
-     * @param value The location to use for the facing.
+     *
+     * @param facting The location to use for the facing.
```
Comment (sixlettervariables) [inline megamek/src/megamek/common/Bay.java]: Speeling.

```suggestion
     * @param facing The location to use for the facing.
```

File: megamek/src/megamek/common/pathfinder/BoardEdgePathFinder.java [hunk 2]
```diff
@@ -136,132 +151,132 @@ private int getAppropriateFacing(Entity entity, int destinationRegion) {
...
     }
-    
+
     /**
      * Helper function that, given a unit facing and a move step, adds turns to the given path until the facing of the path matches
      * the facing of the step.
-     * @param facing
-     * @param path
+     * @param initialPath
+     * @param interesectionStep
```
Comment (sixlettervariables) [inline megamek/src/megamek/common/pathfinder/BoardEdgePathFinder.java]:
```suggestion
     * @param intersectionStep
```

**Comment by HammerGS** [comment]
This pull request **fixes 104 alerts** when merging 5c69c0d5c60256466138feeff5535252b7f7c32e into 54180d578d581886930b45341580a188d2fdeb2e - [view on LGTM.com](https://lgtm.com/projects/g/MegaMek/megamek/rev/pr-066ac2aecf2fc04b4cb8bacc13a06926205474da)

**fixed alerts:**

* 103 for Spurious Javadoc @param tags
* 1 for Missing space in string literal

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**